common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -278,6 +280,31 @@ public class RssBaseConf extends RssConf {
           .defaultValue(16)
           .withDescription("start server service max retry");
 
+  /* Serialization */
```

Comment:
Could you add a document for these config options?

Reply:
After remove kryo, only config 'rss.io.serializations' remained. For now, only one value here. Users do not have to configure it. So I don't think it is necessary to add client_guide.md or server_guide.md temporarily.

For remote merge, some parameters are indeed introduced, I want to create a separate PR to describe it. 
For new users, just need to enable this feature by the one config.

Reply:
If this could be specified in client side, it wil be better. Anyway, this is an initial PR, now this is good for me.